packages/@romejs/cli-flags/Parser.ts

```diff
@@ -599,30 +606,146 @@ export default class Parser<T> {
 
     // Execute all command defineFlags. Only one is usually ran when the arguments match the command name.
     // But to generate autocomplete we want all the flags to be declared for all commands.
+
     const flags = this.getFlagsConsumer();
     for (const command of this.commands.values()) {
       // capture() will cause diagnostics to be suppressed
       const {consumer} = flags.capture();
       await this.defineCommandFlags(command, consumer);
     }
 
-    // this.declaredFlags contains flag information, ignore the keys as they will have command suffixes
-    // utilize `command` for the owned command and `name` for the actual flag name
-
-    // this.commands contains command information
-
-    // reporter.logAllNoMarkup to output to stdout
-    reporter;
+    const prg = process.env.ROME_DEV ? 'dev-rome' : 'rome';
```

Comment:
We want the `cli-flags` package to be usable by other tools.

```suggestion
    const prg = this.options.programName;
```

If we really want to support `dev-rome` then we should set `programName` in `packages/@romejs/cli/cli.ts` and have the `ROME_DEV` environment variable set by the `dev-rome` script.